### PR TITLE
Add support for proxies from text

### PR DESCRIPTION
**Pull request by Risewill23**

### CODE REVIEW

**Comment by thoroc** [comment]
What value does it add compared to specifying your proxies through a list in the `config.json`?

**Comment by Risewill23** [comment]
it's not really compared to anything it just enabling if nothing is found in the json

File: main.py [hunk 1]
```diff
@@ -77,10 +79,19 @@ def __init__(self, config_path):
 
     """ Utils """
 
+    def GetProxiesTXT(self):
```
Comment (thoroc) [inline main.py:82]: please keep the method name as snake case where possible

**Comment by thoroc** [comment]
So that's assuming the user would be dumping a `txt` file from an unspecified website with a list of proxies?

**Comment by thoroc** [comment]
btw you need to run black, on this I think: `nox -rs black`

**Review by thoroc — APPROVED** [review]

**Pull request merged**